e2e/utils/logout.ts

```diff
@@ -0,0 +1,13 @@
+import { Page, expect } from '@playwright/test';
+
+export async function signout(page: Page) {
+  await page.goto('/');
+
+  const cookies = await page.context().cookies();
+  for (const cookie of cookies) {
+    await page.context().clearCookies(cookie);
+  }
+
+  await page.reload();
+  await expect(page.getByRole('link', { name: 'Sign in' })).toBeVisible();
+}
```

Comment:
```suggestion
export async function signout(page: Page) {
  await page.context().clearCookies({ name: 'jwt_access_token' });
}
```
If you just want to clear the auth cookie, this is all you need.